POEM_052.md

```diff
@@ -267,23 +368,27 @@ The matrix vector product method method signature will expect three additional a
 def jac_vec_some_func(x, y, z, d_inputs, d_outputs, mode):
     ...  
 
+@om.func.compute_jcavec_product(jac_vec_some_func)
```

Comment:
having looked at your suggestions, and my own first cut. I think we're both wrong. 

The declare partials annotations should be attached to the `compute_partials` function itself. The primary engineering function shouldn't need to know anything other than "here is your derivative function".